dashboard/app/controllers/lti_v1_controller.rb

```diff
@@ -214,6 +214,50 @@ def sync_course
     end
   end
 
+  # POST /lti/v1/create_integration
+  # Creates a new LtiIntegration
+
+  def create_integration
+    # does this method fire the email upon succesful creation?
+
+    # TODO Nick: These param names sholud be vetted
+    client_id = params[:client_id]
+    admin_email = params[:email]
+    platform_name = params[:lms]
+
+    if platform_name.blank?
```

Comment:
You could also check to make sure the value is in the keys of `Policies::Lti::LMS_PLATFORMS`. Also instead of the `blank?` check I think you can just do `params.require`.

Reply:
Ah cool, yeah I'll refactor this check.